tests/python/gaia-ui-tests/gaiatest/tests/functional/test_wifi_multiple_connect.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.marionette_test import parameterized
+
+from gaiatest import GaiaTestCase
+
+
+class TestWiFiMultipleConnect(GaiaTestCase):
+
+    @parameterized("1st_run")
+    @parameterized("2nd_run")
+    # The second run is necessary to check for the regression bug this
+    # would cause, namely an unusable device, see bug 1190791.
+    def test_connect_and_forget_all_networks(self):
+        for x in range(0, 30):
```

Comment:
30 is a magic number. I'd also comment on the reason why we have this value, and not another like 10